src/core/error.ts

```diff
@@ -13,7 +13,7 @@ import { type ApiError, type ResponseParameters } from "../types.ts";
  */
 export class GrammyError extends Error implements ApiError {
     /** Flag that this request was unsuccessful. Always `false`. */
-    public readonly ok: false = false;
+    public readonly ok = false;
```

Comment:
This is confusing because one needs to check the type hierarchy in order to understand that type widening is being prevented. Please revert.